vignettes/datatable-benchmarking.Rmd

```diff
@@ -2,18 +2,26 @@
 title: "Benchmarking data.table"
 date: "`r Sys.Date()`"
 output:
-  markdown::html_format:
+  litedown::html_format:
     options:
       toc: true
       number_sections: true
     meta:
       css: [default, css/toc.css]
 vignette: >
   %\VignetteIndexEntry{Benchmarking data.table}
-  %\VignetteEngine{knitr::knitr}
+  %\VignetteEngine{litedown::vignette}
   \usepackage[utf8]{inputenc}
 ---
 
+```{r, echo = FALSE, message = FALSE}
+library(data.table)
+opts = litedown::reactor()
+opts$comment = "# "
+opts$error = FALSE
+opts$print = function(x, ...) capture.output(if (isS4(x)) methods::show(x, ...) else print(x, ...))
```

Comment:
[`error = FALSE` will suppress errors in litedown](https://yihui.org/litedown/#sec:option-error), which is probably not what you want. I think you wanted the default `error = NA` instead, so there is no need to set it.

`print = NA` tells litedown to use R's default print function.

```suggestion
litedown::reactor(comment = "# ", print = NA)
```